src/Components/Dialogs/dialogs.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import s from './dialogs.module.css'
+
+const Dialogs = () => {
+  return (
+    <div className={s.profile}>
+ Dialogs've been here
```

Comment:
I think there's a problem with indentation.

Reply:
i added eslint config and prettier .And now it looks like a better.